src/module.c

```diff
@@ -2843,22 +2911,41 @@ int SpellCheckCommandHandler(RedisModuleCtx *ctx, RedisModuleString **argv, int
   return REDISMODULE_OK;
 }
 
+static int CommandIndexPos(RedisModuleString *cmd) {
```

Comment:
Need to make sure the overhead is negligible in benchmarks

Reply:
You think?
I can run them no problem, but I doubt this will be noticed